src/rust/en.tcbscans/res/source.json

```
@@ -3,7 +3,7 @@
 		"id": "en.tcbscans",
 		"lang": "en",
 		"name": "TCB Scans",
-		"version": 3,
-		"url": "https://tcb-backup.bihar-mirchi.com"
+		"version": 4,
+		"url": "https://tcbscans.com"
```

Suggested on discord
```suggestion
		"url": "https://tcbscans.me"
```